### DISCUSSION TITLE
I want to modify the report name？

### DISCUSSION THREAD

**Discussion by liutianping0326** [allure-framework/allure2#1522] [Questions & Support] [answered]
![image](https://user-images.githubusercontent.com/40854706/140450426-c67130c4-25d0-4423-9e62-b8e2cbab950f.png)
![image](https://user-images.githubusercontent.com/40854706/140450461-f7e21777-ceba-46c9-9d75-b31508b365cb.png)
![image](https://user-images.githubusercontent.com/40854706/140450492-3f8ac37a-aa66-431b-8099-299265a4f76e.png)

The report generated by the "jenkins-allure" plug-in does not work after modifying the title in the "summary.json" file

**Comment by baev** [comment, marked as answer]
Fixed in https://github.com/allure-framework/allure2/pull/2229 , released in 2.26.0
